### PR TITLE
refactor(lidar_apollo_instance_segmentation)!: fix namespace and directory structure

### PR DESCRIPTION
**Pull request by ktro2828**
## Description

This PR puts headers in the `autoware` namespace.

Additional works
1. Align directory structure to follow [the coding guidelines.](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#exporting-a-composable-node-component-executables)
2. [Clean unused dependencies](https://github.com/autowarefoundation/autoware/issues/3468)   [LIST](https://github.com/autowarefoundation/autoware.universe/pull/3606)

## Related links

**Parent Issue:**

- Part of: https://github.com/autowarefoundation/autoware/issues/4569

## How was this PR tested?

I tested with 2 types of launcher
1. `ros2 launch autoware_lidar_apollo_instance_segmentation lidar_apollo_instance_segmentation.launch.xml`
2. `ros2 launch autoware_launch logging_simulator.launch.xml` specifying `lidar_detection_model:=apollo`.

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Comment by MasatoSaeki** [comment]
@ktro2828 
I think you need to change `<exec_depend>lidar_apollo_instance_segmentation</exec_depend>` in `autoware.universe/launch/tier4_perception_launch/package.xml`.
And Could you resolve conflict to check this PR?

**Comment by ktro2828** [comment]
> @ktro2828 I think you need to change `<exec_depend>lidar_apollo_instance_segmentation</exec_depend>` in `autoware.universe/launch/tier4_perception_launch/package.xml`. And Could you resolve conflict to check this PR?

@MasatoSaeki Thanks! I've fixed by https://github.com/autowarefoundation/autoware.universe/pull/7995/commits/d731699c1bdcae7e710c07e1537d20c2b69c745b.

**Review by YoshiRi — APPROVED** [review]
LGTM

**Review by MasatoSaeki — APPROVED** [review]
@ktro2828 
Thank you for fixing.
I confirmed the node works without dying

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -126,7 +126,9 @@ perception/elevation_map_loader/** kosuke.takeuchi@tier4.jp shintaro.tomie@tier4
 perception/autoware_euclidean_cluster/** dai.nguyen@tier4.jp yukihiro.saito@tier4.jp
 perception/autoware_ground_segmentation/** abrahammonrroy@yahoo.com dai.nguyen@tier4.jp shunsuke.miura@tier4.jp yoshi.ri@tier4.jp yukihiro.saito@tier4.jp
 perception/autoware_image_projection_based_fusion/** dai.nguyen@tier4.jp koji.minoda@tier4.jp kotaro.uetake@tier4.jp shunsuke.miura@tier4.jp tao.zhong@tier4.jp yoshi.ri@tier4.jp yukihiro.saito@tier4.jp
-perception/lidar_apollo_instance_segmentation/** yoshi.ri@tier4.jp yukihiro.saito@tier4.jp
+perception/autowar_lidar_apollo_instance_segmentation/** yoshi.ri@tier4.jp yukihiro.saito@tier4.jp
```
Comment (yukkysaito) [inline .github/CODEOWNERS:129]: typo
image

**Pull request merged**